packages/devtools_app/test/vm_developer/object_inspector/vm_code_display_test.dart

```diff
@@ -39,15 +39,27 @@ void main() {
         }
       };
       final offset = pow(2, 20) as int;
+      const addressCount = 1000;
       testCode.disassembly = Disassembly.parse(<Object?>[
-        for (int i = 0; i < 1000; ++i) ...[
+        for (int i = 0; i < addressCount; ++i) ...[
           (i * 4 + offset).toRadixString(16),
           'unknown',
           'noop',
           null,
         ]
       ]);
 
+      final ticksTable = CpuProfilerTicksTable.parse(
+        sampleCount: 1000,
```

Comment:
addressCount instead of 1000?

Reply:
The sample count doesn't need to match the address count as the sample count is the number of samples collected throughout the entire process, not just those collected in the context of this piece of code.